src/smartAuthorizationHelper.ts

```diff
@@ -139,7 +158,9 @@ export function hasAccessToResource(
 ): boolean {
     return (
         hasSystemAccess(usableScopes, sourceResource.resourceType) ||
-        (patientOrgs && hasReferenceToResource(patientOrgs, sourceResource, apiUrl, fhirVersion)) ||
+        (patientOrgs &&
+            patientOrgs.length &&
```

Comment:
I think we should be able to assert that patientOrgs is an array. I don't think we require this check.

Reply:
sounds good. We would at least need to check if `patientOrgs` is empty or not.

Reply:
The length check here is check it is not empty? The `hasReferenceToPatientOrgs` already handle empty arrays by returning false. Although this might perform better than calling `hasReferenceToPatientOrgs` with an empty array, that should be the exception rather than the norm. If patientOrgs is present there should be at least one Org reference specified.

Reply:
I am ok with leaving this as-is, but I think the length check here is not necessary.